smak/Server/User.cpp

```diff
@@ -5,9 +5,8 @@
 #include <thread>
 #include "User.h"
 
-smak::User::User(std::shared_ptr<smak::tcpUserSocket> session, const std::string &uName) {
+smak::User::User(std::shared_ptr<smak::tcpUserSocket> session) {
```

Comment:
With what we talked about on the phone, it seems we should put uname back in with its default, and also add in password with a default.